components/NavigationMenu/index.jsx

```diff
@@ -0,0 +1,50 @@
+const NavigationMenu = () => (
+  <>
+    <nav
+      aria-label="Menu Principal"
+      id="visibleDesktopMenu"
+      className="VisibleMenu FlexEndPositioning"
+    >
+      <ul role="menu" className="NavList">
+        <li role="menuitem" className="DisplayNoneOnSmallScreen">
+          <a href="/cathocomponents">Catho Components</a>
+        </li>
+        <li role="menuitem" className="DisplayNoneOnSmallScreen">
+          <a href="/quantum">Quantum</a>
+        </li>
+      </ul>
+    </nav>
+    <input
+      type="checkbox"
+      aria-controls="hiddenDesktopMenu"
+      aria-expanded="false"
+      aria-label="Menu"
+      id="menuSandwichButton"
+    />
+    <label className="SandwichButton" htmlFor="menuSandwichButton">
+      &#x2630;
+    </label>
```

Comment:
It could be a component called SandwichButton. If done, place it in a folder called SubComponents within the Header